frontend/src/components/common/text-input/text-input.tsx

```diff
@@ -0,0 +1,38 @@
+import clsx from 'clsx';
+import { InputType, InputColor } from 'common/enums';
+import { InputChangeCallback } from 'common/types';
+import * as React from 'react';
+import styles from './styles.module.scss';
+
+interface Props {
+  type: InputType;
+  color: InputColor;
+  label: string;
+  name: string;
+  value: string;
+  placeholder?: string;
+  hasError?: boolean;
+  isDisabled?: boolean;
+  onChange: InputChangeCallback;
+}
+
+const TextInput: React.FC<Props> = ({type, color, isDisabled, label, name, value, placeholder, hasError, onChange}) => (
+  <label className={styles.label}>
+    <p className={styles.labelText}>{label}</p>
```

Comment:
```suggestion
    <span className={styles.labelText}>{label}</span>
```
label is a string tag. It can't contain any block elements